cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMaker2AssemblyOperator.java

```diff
@@ -371,11 +373,23 @@ private static String addTLSConfigToMirrorMaker2ConnectorConfig(Map<String, Obje
         return securityProtocol;
     }
 
+    static class ConnectorsComparator implements Comparator<Map<String, Object>>, Serializable {
```

Comment:
I wonder if the name properly describes what it does. All it compares is the names of the connectors. Not the connectors them self - for example their settings etc. If nothing else, this should have some javadoc describing what it does in more detail.

Maybe we could also move the class to the end of the file? I find it a bit weird to have it in the middle.